tests/default/main.tf

```diff
@@ -3,23 +3,147 @@ provider "aws" {
   region  = "us-west-2"
 }
 
+locals {
+  tags = {
+    Environment     = "Test"
+    ServiceProvider = "Rackspace"
+    Terraform       = "true"
```

Comment:
I would remove the `Environment` and `ServiceProvider` tags.  They are already set within the module, and might cause confusion for folks down the road.